gnocchi/storage/__init__.py

```diff
@@ -226,8 +218,13 @@ def get_measures(self, metric, granularities,
             agg_timeseries = list(map(lambda agg: agg.resample(resample),
                                       agg_timeseries))
 
-        return list(itertools.chain(*[ts.fetch(from_timestamp, to_timestamp)
-                                      for ts in agg_timeseries]))
+        return {
```

Comment:
do we really need to change the output? there isn't really a scenario which allows us to get multiple aggregation_methods at once.

Reply:
…yet :)

Reply:
mysterious...